web/js/holidayManagement.js

```diff
@@ -430,6 +430,52 @@ var app = new Vue({
         filterUser(event) {
             this.autocompleteIsActive = true;
             this.usersList = this.users.filter(user => user.name.toLowerCase().includes(event.target.value.toLowerCase()));
+        },
+        toggleDisableButtons() {
+            this.$refs.syncBtn.disabled = !this.$refs.syncBtn.disabled;
+            this.$refs.saveBtn.disabled = !this.$refs.saveBtn.disabled;
+        },
+        syncCalendar: async function () {
+            const url = `services/syncCalendar.php?init=${formatDate(this.init)}&end=${formatDate(this.end)}`;
+            const ranges = this.ranges.map(range => {
+                if (range.coveredDates) {
+                    const end = addDays(range.dates.end, 1);
+                    return {
+                        start: formatDate(range.dates.start),
```

Comment:
I'm getting this error, and the request is never sent to the web service:
```
TypeError: date is undefined
    formatDate http://localhost/phpreport/web/js/holidayManagement.js:23
    ranges http://localhost/phpreport/web/js/holidayManagement.js:444
    syncCalendar http://localhost/phpreport/web/js/holidayManagement.js:440
    VueJS 30
    <anonymous> http://localhost/phpreport/web/js/holidayManagement.js:66

Reply:
Thanks for catching! This was hapening for partial leaves. I just committed a patch.